### PR TITLE
Add PyYAML as indirect requirement

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/requirements.rst [hunk 1]
```diff
@@ -8,12 +8,26 @@ components installed on your system:
...
 * **Git**
 * **Docker** >= 17.05
-* **Docker-compose** >= 1.9
-* **Python** >= 3.5
+* **Python** >= 3.5, with ``pip``
 * **Apache** >= 2.4 (optional, can be used as a front for SSL)
 * **PostgreSQL** >= 9.1/**PostGIS** >= 2.1
 
 
+Required Python package
```
Comment (jwkaltz) [inline doc/integrator/requirements.rst]: package -> packages

File: doc/integrator/requirements.rst [hunk 2]
```diff
@@ -8,12 +8,26 @@ components installed on your system:
...
+* ``PyYAML``
+* ``docker-compose``
+
+They can be install on user dir with:
+
+.. code: bash
+
+   python3 -m pip install --user PyYAML docker-compose
+
+And ``$HOME/.local/lib/bin`` should be in the path.
```
Comment (jwkaltz) [inline doc/integrator/requirements.rst]: "If not yet done, add ... to your path"

File: doc/integrator/requirements.rst [hunk 3]
```diff
@@ -8,12 +8,26 @@ components installed on your system:
...
 * **PostgreSQL** >= 9.1/**PostGIS** >= 2.1
 
 
+Required Python package
+~~~~~~~~~~~~~~~~~~~~~~~
+
+* ``PyYAML``
+* ``docker-compose``
+
+They can be install on user dir with:
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply doc/integrator/requirements.rst]: +1 :-)

**Comment by sbrunner** [comment]
Thanks @jwkaltz

**Pull request merged**